src/main/java/com/example/springboot/controller/BookController.java

```diff
@@ -0,0 +1,85 @@
+package com.example.springboot.controller;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.springboot.dao.IBookDAOService;
+import com.example.springboot.entity.Book;
+
+/**
+ * This is class that control work of application, process requests where URL start with '/book'
+ * @author Kohut Dmytro
+ * @version 1.0
+ */
+@RestController
+@RequestMapping("/book")
+public class BookController {
+	
+	@Autowired
+	private IBookDAOService iBookService;
+	
+	private Map<String, Integer> map = new HashMap<>();
```

Comment:
`map` ? just `map`? Map for what? Why we need the variable? Name of variable should be informed